JitsiConference.js

```diff
@@ -1086,7 +1086,7 @@ JitsiConference.prototype.isModerator = function() {
  */
 JitsiConference.prototype.lock = function(password) {
     if (!this.isModerator()) {
-        return Promise.reject();
+        throw new Error('You are not moderator.');
```

Comment:
This changes the behavior of the API which is not desired. Just pass the error when rejecting the promise ...